src/components/DataDetails.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+'use strict'
+
+import React, { Component, PropTypes } from 'react'
+import { View, Button, Text } from 'react-native'
+import AppStyles from '../styles'
+
+export default class DataDetails extends Component {
+  static propTypes = {
+    text: PropTypes.string.isRequired,
+    onNextButtonPressed: PropTypes.func.isRequired
+  }
+
+  render () {
+    return (
+      <View style={AppStyles.view}>
+        <Text style={AppStyles.hintText}>Data to sign</Text>
+        <Text style={AppStyles.valueText}>{this.props.text}</Text>
+        <Button
+          onPress={this.props.onNextButtonPressed}
+          title='Next'
+          color='gree'
+          accessibilityLabel='Enter Pin'
```

Comment:
I thought that `accessibilityLabel` is completely different than `title` and it's not relevant on that screen, but on the second though it might actually be correct.